apps/roam/src/utils/initializeObserversAndListeners.ts

```diff
@@ -100,6 +101,39 @@ export const initObservers = async ({
     render: (b) => renderQueryBlock(b, onloadArgs),
   });
 
+  const generateTagColorScheme = (hexColor: string) => {
+    const base = tinycolor(hexColor);
+    const hsl = base.toHsl();
+
+    const backgroundColor = tinycolor({
```

Comment:
We should be using an `npm` package to generate a WCAG compliant color based on the `hexColor`.  It looks like this is just setting a static color based on the initial `hexColor`, is that correct?